go.mod

```diff
@@ -0,0 +1 @@
+module github.com/jacobsa/fuse
```

Comment:
I have declared a go module separately in commit https://github.com/jacobsa/fuse/commit/9c4c16288f886ca7266061ed859bcef7b26c22a8, so please remove this addition from your commit :)

Reply:
Removed, thank you.